src/main/java/com/wolox/training/models/Book.java

```diff
@@ -0,0 +1,117 @@
+package com.wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    @Column(nullable = true)
```

Comment:
If you're only going to define the non-nullability, don't use @Column but instead use @NotNull. Change it for every case.
Also, leave a blank line between variables.

Reply:
Thanks, that's a good tip!